Assets/Scripts/EndCombatUI.cs

```diff
@@ -1,16 +1,50 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using TMPro;
 
 public class EndCombatUI : MonoBehaviour {
 
+    public TextMeshProUGUI winner;
+    public TextMeshProUGUI[] playersStats;
+
     public GameObject panel;
 
+    const int MAX_PLAYERS = 4;
+
     public void Show() {
         panel.SetActive(true);
     }
 
     public void Hide() {
         panel.SetActive(false);
     }
+
+    public void UpdateStats(HumanPlayer[] humanPlayers) {
+        float bestScore = -1 * Mathf.Infinity;
+        int winningPlayerIndex = -1;
+
+        for (int i = 0; i < humanPlayers.Length; i++) {
+            if (humanPlayers[i] == null) {

```

Comment:
What happens if we just skip the players that are null?

Reply:
Currently, it would show scores for them (0 kills 0 deaths). Later, I think it will make less sense b/c they won't have a selected character image to show, etc.